src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,12 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
```

Comment:
what is the purpose of this fragment?

Reply:
Create the necessary markup for each element of the array.